apps/deploy-web/src/utils/dateUtils.ts

```diff
@@ -54,3 +42,17 @@ export function getPrettyTime(timeMs: number): string {
     return `${Math.floor(timeMs / 1_000 / 60 / 60)}h ${roundDecimal(timeMs / 1_000 / 60, 2) % 60}m`;
   }
 }
+
+export function createDateRange(input: { from?: Date; to?: Date } = {}): { from: Date; to: Date } {
```

Comment:
just for discussion at this point:
- why `to` defaults to `now`? why not `from`? 
- why default range is 30 days?
- perhaps it makes sense to make the type smth like this for more consistency

```suggestion
export function createDateRange(input: [date1?: Date, date2: Date]): { from: Date; to: Date } {
```